libraries/AP_Compass/AP_Compass_MMC5xx3.cpp

```diff
@@ -86,17 +93,18 @@ bool AP_Compass_MMC5XX3::init()
 
```

Comment:
minimum startup time after software reset on mmc5983 is 10ms, update comment and delay